### PR TITLE
Fix prettier config in VSCode Extension

### PR DESCRIPTION
**Pull request by tbezman**
Some of the config brought over in #3953 borked some stuff.

I "fixed" the config and re ran prettier against the files.

### CODE REVIEW

File: vscode-extension/package.json [hunk 1]
```diff
@@ -137,10 +137,8 @@
     "arrowParens": "avoid",
     "bracketSameLine": true,
     "bracketSpacing": false,
-    "requirePragma": true,
```
Comment (tbezman) [inline vscode-extension/package.json:140]: I don't think we want this here, it seems tedious and unnecessary.

File: vscode-extension/package.json [hunk 2]
```diff
@@ -137,10 +137,8 @@
...
     "singleQuote": true,
-    "trailingComma": "all",
-    "parser": "flow"
```
Comment (tbezman) [inline vscode-extension/package.json:143]: Let prettier infer the language.

File: vscode-extension/package.json (hunk 1 above)
Comment (captbaritone) [reply vscode-extension/package.json:140]: Given that this is TS, I'd agree.

File: vscode-extension/package.json (hunk 2 above)
Comment (captbaritone) [reply vscode-extension/package.json:143]: Oh right. Thanks!

**Review by captbaritone — APPROVED** [review]
Just one nit

**Pull request closed without merging**